analysis/civic/examples/transform/civic_cdm_20211227.json

```diff
@@ -5,7 +5,7 @@
             "description": "A phase III clinical trial (NCT00949650) found that median progression free survival among patients with exon 19 deletions or L858R EGFR mutations (n = 308) was 13.6 months for afatinib and 6.9 months for chemotherapy (HR, 0.47; 95% CI, 0.34 to 0.65; P = 0.001).",
             "direction": "supports",
             "evidence_level": "civic.evidence_level:B",
-            "proposition": "proposition:127",
+            "proposition": "proposition:5k7StVvnNBf6qxTgXgZBCSaiyWScvS3Q",
```

Comment:
I assume this will be removed in the proposition / document issue?

Reply:
??? hold the phone

Reply:
I think this was happening because the CIViC examples generator just uses the first 5 assertions, and this one ^ no longer comes up in that list (so it was left over from my work on another branch). I cleared out my working folders and reran everything and it should be a bit less weird now